endpoints/openrtb2/video_auction.go

```diff
@@ -380,12 +380,8 @@ func max(a, b int) int {
 	return b
 }
 
-func createImpressionTemplate(imp openrtb.Imp, video *openrtb.Video) openrtb.Imp {
-	imp.Video = &openrtb.Video{}
-	imp.Video.W = video.W
-	imp.Video.H = video.H
-	imp.Video.Protocols = video.Protocols
-	imp.Video.MIMEs = video.MIMEs
+func createImpressionTemplate(imp openrtb.Imp, video openrtb.Video) openrtb.Imp {
+	imp.Video = &video
```

Comment:
Do you think it would be cleaner to leave the method signature the same as it was before and make the video copy within the method? That might be easier to read / understand.

Reply:
Refactored.